packages/twenty-zapier/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "twenty-ui",
```

Comment:
twenty-zapier!

Let's start the version at 1.0.0 for now!

Reply:
ok